dojo/tools/checkmarx_osa/__init__.py

```diff
@@ -0,0 +1 @@
+__author__ = 'SPoint42'
```

Comment:
```suggestion
```
Meta attribute __author__ are useless as we are tracking authorship with our VCS.